build.sbt

```diff
@@ -4,6 +4,8 @@ import Dependencies._
 addCommandAlias("fmt", "; scalafmtAll; scalafmtSbt")
 addCommandAlias("fmtCheck", "; scalafmtCheckAll; scalafmtSbtCheck")
 
+addCommandAlias("prePR", "fmt")
```

Comment:
Should we add test into that list? And maybe bench/compile?

Reply:
yup let's do it!